grantnav/frontend/templatetags/frontend.py

```diff
@@ -64,7 +64,9 @@ def flatten_dict(data, path=tuple()):
              "fundingOrganization: id_and_name", "recipientLocation",
              "awardDateDateOnly", "plannedDates: endDateDateOnly",
              "plannedDates: startDateDateOnly",
-             "additional_data_added", "title_and_description"
+             "additional_data_added", "title_and_description",
+             # https://github.com/ThreeSixtyGiving/grantnav/issues/795
```

Comment:
Not a blocker but don't think there is a need for that as a comment

Reply:
Suppose it is linked in the commit message so yeah, but I'll leave it now